VocaDb.ResXFileCodeGenerator.Tests/GeneratorTests.cs

```diff
@@ -390,12 +393,123 @@ public static class CommonMessages
 				CustomToolNamespace: null,
 				ClassName: "CommonMessages",
 				PublicClass: true,
-				NullForgivingOperators: false
+				NullForgivingOperators: false,
+        StaticClass: true
 			)
 		);
 		source.Should().Be(expected);
 	}
 
+  [Fact]
+  public void Generate_StringBuilder_Name_NotValidIdentifier()
```

Comment:
Can you please create a folder something like `github-issues/3` and move this test into it?

Reply:
Refactored to GithubIssues/Issue3/GeneratorTests.cs for this test.